src/main/java/com/microsoft/sqlserver/jdbc/SQLServerMSAL4JUtils.java

```diff
@@ -41,7 +57,12 @@ class SQLServerMSAL4JUtils {
 
     static final String REDIRECTURI = "http://localhost";
     static final String SLASH_DEFAULT = "/.default";
-    static final String ACCESS_TOKEN_EXPIRE = " access token expires on the following date: ";
+    static final String ACCESS_TOKEN_EXPIRE = "access token expires: ";
+
+    private final static String msalVersion = com.microsoft.aad.msal4j.PublicClientApplication.class.getPackage()
+            .getImplementationVersion();
+
+    private final static String logContext = "MSAL version " + msalVersion + ": ";
```

Comment:
Naming convention. `logContext` has 19 other usages that would also need to be fixed.
```suggestion
    private final static String MSAL_VERSION = com.microsoft.aad.msal4j.PublicClientApplication.class.getPackage()
            .getImplementationVersion();

    private final static String LOG_CONTEXT = "MSAL version " + MSAL_VERSION + ": ";
```